frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -554,6 +570,8 @@ const CreateBackingStoreForm: React.FC<CreateBackingStoreFormProps> = withHandle
         ...bsPayload.spec['s3Compatible'],
         endpoint: providerDataState.endpoint,
       };
+    } else if (provider === BC_PROVIDERS.IBM) {
+      bsPayload.spec.ibmCos = { ...bsPayload.spec.ibmCos, endpoint: providerDataState.endpoint };
```

Comment:
`ibmCos` -> `ibmCOS`

Reply:
I feel these names are predefined by CRD, hence it can't be renamed.

Reply:
@gnehapk could you please reiterate the second comment?

Reply:
> `ibmCos` -> `ibmCOS`

This is defined in the Noobaa operator, we can't come up with our own naming convention.

Reply:
Resolving as this suggestion is not applicable.